src/api/requestsApi.ts

```diff
@@ -116,23 +125,25 @@ export const useRequest = (requestId: number) => {
   });
 };
 
-const getRequests = async () => {
-  // Grab all fields, and expand Person fields
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This returns a generic type, of your PagedRequest[]
PagedItemCollection<PagedRequest[]>

Reply:
I was seeing you had defined PagedRequest -- and in my head I was thinking that is what is from SP (Input) -- but as your comment pointed out -- that is the Output -- if you do define the results from SharePoint, like you did with SPDcoument, you can pass the type to getPaged<TypeHere>() -- and then it returns that type type wrapped in the PagedItemCollection<T> generic